site/plugins.json

```diff
@@ -429,7 +429,17 @@
     "name": "JavaScript obfuscator",
     "package": "netlify-plugin-js-obfuscator",
     "repo": "https://github.com/iamanishroy/netlify-plugin-js-obfuscator",
-    "version": "1.0.20"
+    "version": "1.0.20",
+    "compatibility": [
```

Comment:
Should this have been committed/pushed? If so, any reason for adding this compatibility req to this specific package?

Reply:
No! This was due to debugging, I'm glad you saw this.